test/services/CategorisationServiceSpec.scala

```diff
@@ -307,6 +313,110 @@ class CategorisationServiceSpec extends SpecBase with BeforeAndAfterEach {
 
       }
 
+      "if NIPHL is not authorised and has NIPHL assessments" in {
```

Comment:
this is not a valid NIPHLs assessment

Reply:
This is for the code coverage. Now removed not needed with new logic.